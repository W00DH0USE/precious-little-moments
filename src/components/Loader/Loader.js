import React from 'react';
import './Loader.css';

function Loader() {
    return (
        <object type="image/svg+xml" >
            Your browser does not support SVG
        </object>
    )
}

export default Loader;