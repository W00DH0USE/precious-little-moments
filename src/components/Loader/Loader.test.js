import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Loader from '../Loader/Loader';


describe(`Loader component`, () => {
    it('renders without crashing', () => {
        const wrapper = shallow(<Loader />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});