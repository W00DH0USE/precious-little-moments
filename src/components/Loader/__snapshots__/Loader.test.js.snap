// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component renders without crashing 1`] = `
<object
  data="output.svg"
  type="image/svg+xml"
>
  Your browser does not support SVG
</object>
`;
